dask_jobqueue/tests/test_pbs.py

```diff
@@ -99,41 +103,93 @@ def test_basic(loop):
             start = time()
             while len(client.scheduler_info()['workers']) > 0:
                 sleep(0.100)
-                assert time() < start + 10
+                assert time() < start + QUEUE_WAIT
 
-            assert not cluster.jobs
+            assert not cluster.running_jobs
 
 
 @pytest.mark.env("pbs")  # noqa: F811
 def test_adaptive(loop):
-    with PBSCluster(walltime='00:02:00', processes=1, threads=2, memory='2GB', local_directory='/tmp',
-                    job_extra=['-V'], loop=loop) as cluster:
+    with PBSCluster(walltime='00:02:00', processes=1, threads=2, memory='2GB',
+                    local_directory='/tmp', job_extra=['-V'],
+                    loop=loop) as cluster:
         cluster.adapt()
         with Client(cluster) as client:
             future = client.submit(lambda x: x + 1, 10)
-            assert future.result(60) == 11
+            assert cluster.pending_jobs or cluster.running_jobs
+            assert future.result(QUEUE_WAIT) == 11
+
+            start = time()
+            while not len(cluster.pending_jobs):
```

Comment:
Don't we risk to directly have cluster.running_jobs non empty here, without seeing the cluster.pending_jobs step?

Reply:
Maybe one of the test failure comes from this.